templates/quickstart-jira-dc-with-vpc.template.yaml

```diff
@@ -420,7 +438,7 @@ Parameters:
     Type: String
   DBStorage:
     Default: 200
-    Description: Database allocated storage size, in gigabytes (GB)
+    Description: Database allocated storage size, in gigabytes (GB). If you choose Provisioned IOPS, storage should be between 100 and 6144. # (not used for Aurora)
```

Comment:
Do we have to change this description until we are ready to merge aurora?

Reply:
Why would we change the description? The Aurora part is commented out